docs/sources/tempo/setup/operator/monitor.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Monitor
+description: Monitor TempoStack instances
+menuTitle: Monitor
+weight: 300
+aliases:
+- /docs/tempo/operator/monitor
+---
+
+# Monitor
+
+Tempo operator and `TempoStack` operands can be monitored.
+The monitoring configuration for operands is exposed the `TempoStack` CR:
+
+```yaml
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+spec:
+  observability:
+    metrics:
+      createServiceMonitors: true
+      createPrometheusRules: true
+    tracing:
+      sampling_fraction: 1.0
+      jaeger_agent_endpoint: localhost:6831
+```
+
+## Distributed Tracing
```

Comment:
Are there other configurations that will be included besides the section for distributed tracing? If not, do we need the distributed tracing  heading? If not, we could move line 30 ("All Tempo components...") underneath line 32 heading (Configure distributed tracing of operands") and change that heading from a level 3 to a level 2. Subsequent headings would all need to come up a level as well. 

What do you think? I'm happy to do this in a comment if it makes sense.

Reply:
There will be a metrics section as well

Reply:
Okay, thank you. That makes sense. I'll undo my suggestions.